tests/test-signed-pull.sh

```diff
@@ -98,13 +98,30 @@ ${CMD_PREFIX} ostree --repo=repo config set 'remote "origin"'.verification-dummy
 ${CMD_PREFIX} ostree --repo=repo config unset 'remote "origin"'.verification-dummy-file
 test_signed_pull "dummy" ""
 
+repo_init
+if ${CMD_PREFIX} ostree --repo=repo remote add other --sign-verify=trustme=inline:ok http://localhost 2>err.txt; then
+    assert_not_reached "remote add with invalid keytype succeeded"
+fi
+assert_file_has_content err.txt 'Requested signature type is not implemented'
+if ${CMD_PREFIX} ostree --repo=repo remote add other --sign-verify=ed25519 http://localhost 2>err.txt; then
```

Comment:
1. Please move all `ed25519` related tests below -- after `has_sign_ed25519` check -- in other case it throws an error if there is no ed25519 support: `error: Requested signature type is not implemented`
2. And as far as you there -- please add the test for `dummy` engine with inlined key -- it should to work everywhere.

Reply:
OK, done.